src/main.ts

```diff
@@ -170,9 +175,9 @@ class AirQ extends utils.Adapter {
 				const serial = sensorsData.SN;
 				const shortID = serial.slice(0,5);
 				this.setState('info.connection', { val: true, ack: true });
+				this.log.info('Air-Q connected.');
```

Comment:
please change all instances to `air-q`

Reply:
i'll assume it's still Air-q if it's the start of a sentence :eyes: @ElmoWater